src/routes/spirit-board/growth.svelte

```diff
@@ -221,12 +245,12 @@
               {#each growthGroup.growthActions as growthAction, k (growthAction.id)}
                 <div class="growth-action-container">
                   <div class="control">
-                    <input
+                    <AutoComplete
                       id={`growthSet${i}Group${j}Action${k}`}
-                      class="input"
-                      type="text"
-                      tabindex="1"
+                      elementType="input"
                       placeholder="Growth Action"
+                      showListImmediately={true}
+                      {validAutoCompleteValues}
                       on:blur={easyReport}
```

Comment:
This on:blur event no longer runs, but I think we could pass a "do this on blur" function to AutoComplete and have it run there.